source/_static/css/swagger-ui/index.css

```diff
@@ -0,0 +1,291 @@
+html {
+    box-sizing: border-box;
+    overflow: -moz-scrollbars-vertical;
+    overflow-y: scroll;
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: inherit;
+}
+
+body {
+    margin: 0;
+    background: #fafafa;
+}
+
+/*********************Invoca Swagger UI Styling************************/
+.swagger-ui table.multiline-table td {
+  vertical-align: top !important;
+}
+
+.swagger-ui table {
+  border-spacing: 2px !important;
+  border-collapse: collapse !important;
+  border-color: grey !important;
+  padding: 8px !important;
+  width: 100% !important;
+}
+
+.swagger-ui th {
+  background-color: #6A6A6A !important;
+  padding: 8px !important;
+  font-size: 10pt !important;
+  color: #E4E4E4 !important;
+  border-color: #595959 !important;
+  border-top-width: 0 !important;
+  border-bottom-width: 0 !important;
+  border-right-width: 1px !important;
+  border-right-style: solid !important;
+  font-weight: bold !important;
+  text-align: left !important;
+}
+
+.swagger-ui .responses-header td {
+  background-color: #6A6A6A !important;
+  padding: 8px !important;
+  font-size: 10pt !important;
+  color: #E4E4E4 !important;
+  border-color: #595959 !important;
+  border-top-width: 0 !important;
+  border-bottom-width: 0 !important;
+  border-right-width: 1px !important;
+  border-right-style: solid !important;
+  font-weight: bold !important;
+  text-align: left !important;
+}
+
+.swagger-ui tr {
+  border-right: 1px solid #d5d5d5 !important;
+  padding: 8px !important;
+  line-height: 1.42857143 !important;
+}
+
+.swagger-ui td {
+  padding: 8px !important;
+  border-right: 1px solid #d5d5d5 !important;
+  font-size: 10pt !important;
+}
+
+.swagger-ui td p {
+  font-size: 10pt !important;
+}
+
+/*
+.swagger-ui tr:nth-child(even) {
+  color: #636060 !important;
+  background-color: #FFF !important;
+}
+
+.swagger-ui tr:nth-child(odd) {
+  color: #4A4A4A !important;
+  background-color: #FFF !important;
+}
+*/
```

Comment:
Did we want to just remove this?

Reply:
I believe so. I think I had kept it around while I was experimenting with simplifying the rules that are now commented out.